content/en/tools/digital-currencies/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+date: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: Digital Currencies
+    weight: 600
+    identifier: digital-currencies
+title: Welcome to Digital Currencies
+description: "Documentation describing the Digital Currencies feature from R3"
+
+---
+
+# Welcome to Digital Currencies
+
+
+R3 Digital Currencies for Corda 5 Overview enables central banks and commercial banks to define their own digital currency token types, mint tokens based on those definitions, and enable transacting entities on the same network to transfer these tokens to each other.
```

Comment:
add : instituitions

Reply:
It now reads:

---

R3 Digital Currencies for Corda 5 enables central banks, commercial banks and institutions to define their own digital currency token types and perform the following actions based on those definitions:

* mint tokens
* request an issuance of tokens from the issuer
* transfer tokens to each other
* redeem tokens
* burn tokens

For more information, see the following topics:

--

The page now also includes/will include hyperlinks to all topics.